Dockerfile

```diff
@@ -48,34 +47,22 @@ RUN \
       # Additional dependencies for python-build
       libbz2-dev \
       llvm \
-      libncursesw5-dev
-
-RUN apt-get install \
-      -qq \
-      --yes \
-      --no-install-recommends \
-      --no-install-suggests \
+      libncursesw5-dev \
+      # Not sure what these dependencies are for
       nodejs \
       npm
```

Comment:
The `nodejs` and `npm` dependencies are for the Node-based scanners in this project (which use `phantomas` as a headless browser). Though I am definitely hesitant to depend on `apt` for that, but that's for another PR.